client/webserver/webserver.go

```diff
@@ -164,7 +177,7 @@ func New(core clientCore, addr, customSiteDir string, logger dex.Logger, reloadH
 	var siteDir string
 	absDir, _ := filepath.Abs("site")
 	for _, dir := range []string{
-		customSiteDir,
+		cfg.CustomSiteDir,
```

Comment:
Another possible follow-up: in the loop body, do a `continue` if `dir == ""`, as is usually the case with the default `CustomSiteDir`.  Otherwise we see something confusing like this:

```
[DBG] WEB: Looking for site in 
[DBG] WEB: Looking for site in /home/user/github/decred/dcrdex/client/cmd/dexc/site
[DBG] WEB: Looking for site in /home/user/github/decred/dcrdex/client/cmd/dexc/site
[DBG] WEB: Looking for site in /home/user/github/decred/dcrdex/client/webserver/site
[INF] WEB: Located "site" folder at /home/user/github/decred/dcrdex/client/webserver/site
```